src/main/g8/build.sbt

```diff
@@ -1,42 +1,82 @@
-ThisBuild / version       := "0.0.1-SNAPSHOT"
+import Dependencies._
+
 ThisBuild / organization  := "$organization;format="lower,package"$"
 ThisBuild / scalaVersion  := "$scala_version$"
 ThisBuild / useSuperShell := false
 
 lazy val `$name;format="norm"$` =
   project
     .in(file("."))
-    .aggregate(entities, core, delivery, persistence, main)
+    .aggregate(domain, core, delivery, persistence, main)
     .settings(
       name := "$name$"
     )
 
-addCommandAlias("run", "main/run")
-
-lazy val entities =
+lazy val domain =
   project
-    .in(file("entities"))
+    .in(file("01-domain"))
+    .settings(commonSettings)
 
 lazy val core =
   project
-    .in(file("core"))
-    .dependsOn(entities % oneToOneClasspathDependencies)
+    .in(file("02-core"))
+    .dependsOn(domain % oneToOneClasspathDependencies)
+    .settings(commonSettings)
+    .settings(commonTestDependencies)
 
 lazy val delivery =
   project
-    .in(file("delivery"))
+    .in(file("03-delivery"))
     .dependsOn(core % oneToOneClasspathDependencies)
+    .settings(commonSettings)
+    .settings(commonTestDependencies)
 
 lazy val persistence =
   project
-    .in(file("persistence"))
+    .in(file("03-persistence"))
     .dependsOn(core % oneToOneClasspathDependencies)
+    .settings(commonSettings)
+    .settings(commonTestDependencies)
 
 lazy val main =
   project
-    .in(file("main"))
+    .in(file("04-main"))
     .dependsOn(delivery % oneToOneClasspathDependencies)
     .dependsOn(persistence % oneToOneClasspathDependencies)
+    .settings(commonSettings)
+
+lazy val commonTestDependencies = Seq(
+  libraryDependencies ++= Seq(
+    com.github.alexarchambault.`scalacheck-shapeless_1.15`,
+    org.scalacheck.scalacheck,
+    org.scalatest.scalatest,
+    org.scalatestplus.`scalacheck-1-15`,
+    org.typelevel.`discipline-scalatest`,
+  ).map(_ % Test),
+)
+
+lazy val commonSettings =
+  compilerPlugins ++ commonScalacOptions ++ Seq(
+    update / evictionWarningOptions := EvictionWarningOptions.empty
+  )
+
+lazy val compilerPlugins = Seq(
+  addCompilerPlugin(com.olegpy.`better-monadic-for`),
+  addCompilerPlugin(org.augustjune.`context-applied`),
+  addCompilerPlugin(org.typelevel.`kind-projector`),
+)
+
+lazy val commonScalacOptions = Seq(
+  Compile / console / scalacOptions := {
+    (Compile / console / scalacOptions)
+      .value
+      .filterNot(_.contains("wartremover"))
+      .filterNot(Scalac.Lint.toSet)
+      .filterNot(Scalac.FatalWarnings.toSet) :+ "-Wconf:any:silent"
+  },
+  Test / console / scalacOptions :=
+    (Compile / console / scalacOptions).value,
+)
 
 lazy val oneToOneClasspathDependencies: String =
```

Comment:
If you `import Util._` it has `Cctt` which does the same thing so you can throw this one out.